internal/protocol/tcp_server.go

```diff
@@ -26,11 +27,13 @@ func TCPServer(listener net.Listener, handler TCPHandler, logf lg.AppLogFunc) {
 			// theres no direct way to detect this error because it is not exposed
 			if !strings.Contains(err.Error(), "use of closed network connection") {
 				logf(lg.ERROR, "listener.Accept() - %s", err)
```

Comment:
If we take the approach I proposed, this log line should communicate that the error is `FATAL`.

Reply:
Done.